sh/src/helm-template

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+#
+# Render chart templates locally using helm template. Display the output objects as a Kubernetes List.
+
+set -eo pipefail
+
+NAME="name"
+CHART_PATH="chart_path"
+USAGE=$(cat << END
+Render chart templates locally using helm template. Display the output objects
+as a Kubernetes List. If piped a Kubernetes List in addition to ${NAME} and
+${CHART_PATH} arguments then render the chart objects into the piped list,
+overwriting any chart objects that already exist in the list.
+
+Configured using a ConfigMap with the following keys:
+
+${NAME}: Name of helm chart.
+${CHART_PATH}: Relative path to helm chart.
+
+Example:
+
+To expand a chart named 'my-chart' at '../path/to/helm/chart':
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image:  gcr.io/kpt-functions/helm-template
+    config.kubernetes.io/local-config: "true"
+data:
+  ${NAME}: my-chart
+  ${CHART_PATH}: ../path/to/helm/chart
+END
+)
+
+err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+}
+
+read_arguments() {
+  if [[ $# -le 1 ]] ; then
+    err "${USAGE}"
+    exit 1
+  fi
+
+  for arg in "$@"; do
+    case $arg in
+      "${NAME}="*) name="${arg#*=}" ;;
+      "${CHART_PATH}="*) chartPath="${arg#*=}" ;;
+      *)
+        err "${USAGE}"
+        exit 1
+        ;;
+    esac
+  done
+}
+
+create_tmp() {
+  tmp=$(mktemp -d "/tmp/helm-template.XXXXXXXX")
+  # If there is input through stdin then write it to tmp
```

Comment:
Since this is meant to run in a pipeline, we will input a list of kubernetes objects through stdin. These objects can be overwritten by helm expand if they have the same name/namespace. Documented this in USAGE.